backend/remote-state/azure/helpers_test.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 const (
-	sasSignedVersion = "2017-07-29"
+	// required for Azure Stack
+	sasSignedVersion = "2015-04-05"
```

Comment:
Isn't this a breaking change? 🤔

Reply:
this feature hasn't shipped yet - but this is a test either way (we're assuming users submit a valid SAS Token)